libyang/context.py

```diff
@@ -468,3 +511,41 @@ def __iter__(self) -> Iterator[Module]:
         while mod:
             yield Module(self, mod)
             mod = lib.ly_ctx_get_module_iter(self.cdata, idx)
+
+    def free_module_data(self, cdata) -> None:
+        self._cdata_modules.remove(cdata)
+
+    def get_module_data(
+        self,
```

Comment:
Please add docstring and describe the arguments.

Reply:
Done, but the example is pretty pointless, as this API is called by libyang directly and not by user. Check the unit test that shows real example how to use this